addons/website/static/src/xml/website.editor.xml

```diff
@@ -37,11 +37,11 @@
         <div class="form-row justify-content-between o_options_container"/>
     </div>
     <t t-name="website.theme_customize_modal_option">
-        <div t-attf-class="o_theme_customize_option my-1 flex-grow-0 #{font ? 'o_theme_customize_option_font_' + font : ''} #{widget ? 'o_theme_customize_with_widget' : ''}">
+        <div t-attf-class="o_theme_customize_option my-1 flex-grow-0 #{font ? 'o_theme_customize_option_font_' + font : ''} #{widget and widget != 'auto' ? 'o_theme_customize_with_widget' : ''}">
             <img t-if="icon" t-att-src="icon"/>
 
             <t t-set="label" t-value="font ? '' : string"/>
-            <t t-if="alone and !widget" t-call="website.components.switch"/>
+            <t t-if="(alone and !widget) or (alone and widget == 'auto' and !isSelectionContainer)" t-call="website.components.switch"/>
```

Comment:
I don't see why `isSelectionContainer` is doing in that condition and your condition can be simplified.